astropy/units/format/cds.py

```diff
@@ -20,13 +20,12 @@
 import os
 import re
 
-
-from .base import Base
-from . import core, utils
 from astropy.units.utils import is_effectively_unity
 from astropy.utils import classproperty, parsing
 from astropy.utils.misc import did_you_mean
 
+from . import core, utils
+from .base import Base
```

Comment:
```suggestion
from .base import Base

```
I think this will fix the flake8 error in the code style CI